contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -25,8 +25,8 @@ Make PostHog the best and most performant product analytics platform.
 
 ## Output metrics
 
+- Primary: [Discoveries](https://app.posthog.com/insights/O4R-Frza?events=%5B%5D&actions=%5B%7B%22id%22%3A%2210784%22%2C%22math%22%3A%22total%22%2C%22name%22%3A%22Discoveries%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%5D&display=ActionsLineGraph&insight=TRENDS&interval=week&date_from=-90d&new_entity=%5B%5D&properties=%5B%5D&breakdown_type&filter_test_accounts=true#fromDashboard=20464)
```

Comment:
I think Eng/Product teams should maybe be focused on Discovers / active users? Otherwise the Marketing activities would have a huge effect here.

Reply:
Also, I would consider just creating a dashboard with all the main small team metrics and linking the dashboard globally, instead of each insight.

Reply:
I think this is a decent point, but I'd rather focus on volume so that if the eng/product teams want to add ie a feature to encourage collaboration / to add more users, to get more discoveries - I'd like them to do so. I agree long term we should do this, perhaps when we get larger we could split out responsibility further